src/frontend/src/pages/HomePage/OnboardingBlock.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box, Typography } from '@mui/material';
+
+interface OnboardingBlockProps {
+  title: string;
+  text: string;
+}
+
+const OnboardingBlock: React.FC<OnboardingBlockProps> = ({ title, text }) => {
```

Comment:
Love the organization for the code and creating a new file for this block. What I think might be a better idea is to use grid to create three sections (And you can leave placeholder comment for the other two blocks like useful links and questions block) and then just add the styling for your text and block there. This way we won't have a different component for the onboarding block, useful links, and questions and instead itll all be in one file